implementation/jwt-auth/src/main/java/io/smallrye/jwt/auth/principal/AbstractKeyLocationResolver.java

```diff
@@ -115,6 +115,7 @@ protected HttpsJwks initializeHttpsJwks(String location)
                     new InetSocketAddress(authContextInfo.getHttpProxyHost(), authContextInfo.getHttpProxyPort())));
         }
         theHttpsJwks.setSimpleHttpGet(httpGet);
+        theHttpsJwks.setRetainCacheOnErrorDuration(authContextInfo.getJwksRetainOnErrorDuration());
```

Comment:
```suggestion
        theHttpsJwks.setRetainCacheOnErrorDuration(authContextInfo.getJwksRetainOnErrorDuration() * 60);
```

It depends on updating the property description that it is a duration in mins (I don't recall now why it is not Duration - probably because it is too flexible - one can set a year by default by mistake etc)